yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/operations/MapperOrReducerSpec.java

```diff
@@ -95,6 +90,42 @@ protected MapperOrReducerSpec(Class<?> mainClazz, Builder<?> builder) {
         prepareTimeLimit = builder.prepareTimeLimit;
     }
 
+    // Copied from `BaseLayerDetector.guess_base_layers`
+    public static StartOperation guessBaseLayers(StartOperation req) {
```

Comment:
We write javadoc for all new methods.

Reply:
What do you want me to write in this javadoc?

Reply:
Don't we want simple unit-test for this code?

Reply:
I don't know, do we? The python version didn't have any.

Reply:
> Why it is static method and not method of the builder?

No specific reason. Should it be non-static method for some reason?